src/main/java/com/icthh/xm/ms/entity/security/filter/EntityFunctionPermissionMsNameFilter.java

```diff
@@ -0,0 +1,25 @@
+package com.icthh.xm.ms.entity.security.filter;
+
+import static org.apache.commons.lang3.StringUtils.isBlank;
+import static org.apache.commons.lang3.StringUtils.isNotBlank;
+import static org.apache.commons.lang3.StringUtils.startsWithIgnoreCase;
+
+import com.icthh.xm.commons.permission.service.filter.PermissionMsNameFilter;
+import org.apache.commons.lang3.StringUtils;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+@Component("permissionMsNameFilter")
+public class EntityFunctionPermissionMsNameFilter implements PermissionMsNameFilter {
+
+    private static final String CUSTOM_PRIVILEGES_SECTION = "entity-functions";
```

Comment:
pls change to `entity-`

Reply:
you mean to `ENTITY_CUSTOM_PRIVILEGES_SECTION` ?

Reply:
@sergeysenja1992 constant value have been updated to `entity-`